advanced-javascript/tests/arrays.test.js

```diff
@@ -18,43 +18,108 @@ describe('Arrays', () => {
   describe('`each`', () => {
     it('should be a function', () => {
       const each = arrayFunctions.each;
-      expect(each).to.be.a('string');
+      expect(each).to.be.a('function');
     });
     // begin here
+    it('should execute call back at least once', () => {
+      const callBack = sinon.spy();
+      const each = arrayFunctions.each;
```

Comment:
Pro Tip: you can use destructuring assignment here!

`const { each } = arrayFunctions;`